addons/mrp/models/stock_move.py

```diff
@@ -76,6 +81,14 @@ def _compute_is_done(self):
         for move in self:
             move.is_done = (move.state in ('done', 'cancel'))
 
+    @api.multi
+    def write(self, vals):
+        if vals.get('product_uom_qty'):
+            for move in self.filtered(lambda x : x.state not in ('done', 'draft')):
+                production = move.production_id or move.raw_material_production_id
+                move.log_message(production, 'mrp.track_production_move_template', vals, {})
+        return super(StockMove, self).write(vals)
```

Comment:
This should only be for pickings normally. (e.g. I don't want plenty of messages when you update the qty on a production order)